docs/conf.py

```diff
@@ -47,7 +47,6 @@
     "sphinx.ext.githubpages",
     "sphinxcontrib.httpdomain",
     "sphinx_tabs.tabs",
-    "sphinx_prompt",
```

Comment:
Is this still required? Also, could you please elaborate on the build issue you faced?